cmd/skupper/skupper.go

```diff
@@ -753,11 +753,7 @@ func NewCmdExpose(newClient cobraFunc) *cobra.Command {
 	cmd.Flags().StringVar(&exposeOpts.ProxyTuning.NodeSelector, "proxy-node-selector", "", "Node selector to control placement of router pods")
 	cmd.Flags().StringVar(&exposeOpts.ProxyTuning.Affinity, "proxy-pod-affinity", "", "Pod affinity label matches to control placement of router pods")
 	cmd.Flags().StringVar(&exposeOpts.ProxyTuning.AntiAffinity, "proxy-pod-antiaffinity", "", "Pod antiaffinity label matches to control placement of router pods")
-	cmd.Flags().BoolVar(&exposeOpts.EnableTls, "enable-tls", false, "If specified, this service will have TLS support for services.")
-
-	// TODO: Disabled flag to enable when the router image version that support TLS is ready for skupper
-	f := cmd.Flag("enable-tls")
-	f.Hidden = true
+	cmd.Flags().BoolVar(&exposeOpts.EnableTls, "enable-tls", false, "If specified, this service will have TLS support towards the router.")
```

Comment:
The description is not that clear to me... perhaps 'If specified, the service will be exposed over TLS'? @pwright @ajssmith any thoughts on wording?

Reply:
"If specified, the service communication will be encrypted using TLS."
Altho, Gordon's alternative is good too.